backend/tests/test_code_services.py

```
@@ -0,0 +1,388 @@
+
+
+@pytest_asyncio.fixture
+async def seed_global_services(db_session: AsyncSession):
+    """Seed global status rows for all supported services."""
+    for key in VALID_SERVICE_KEYS:
+        db_session.add(CodeServiceGlobalStatus(service_key=key, is_active=True))
+    await db_session.commit()
+
+
+@pytest_asyncio.fixture
+async def master_headers(client: AsyncClient, master_user: User) -> dict:
```

Ruff (0.15.14)

[warning] Unused function argument: `master_user`

(ARG001)